fiberi18n/i18n.go

```diff
@@ -63,42 +61,48 @@ func (c *Config) initLocalizerMap() {
 	if _, ok := localizerMap.Load(lang); !ok {
 		localizerMap.Store(lang, i18n.NewLocalizer(c.bundle, lang))
 	}
+	c.mu.Lock()
 	c.localizerMap = localizerMap
+	c.mu.Unlock()
 }
 
 /*
-MustGetMessage get the i18n message without error handling
+MustLocalize get the i18n message without error handling
 
 	  param is one of these type: messageID, *i18n.LocalizeConfig
 	  Example:
-		MustGetMessage("hello") // messageID is hello
-		MustGetMessage(&i18n.LocalizeConfig{
+		MustLocalize(ctx, "hello") // messageID is hello
+		MustLocalize(ctx, &i18n.LocalizeConfig{
 				MessageID: "welcomeWithName",
 				TemplateData: map[string]string{
 					"name": context.Param("name"),
 				},
 		})
 */
-func MustGetMessage(params interface{}) string {
-	message, _ := GetMessage(params)
+func MustLocalize(ctx *fiber.Ctx, params interface{}) string {
+	message, err := Localize(ctx, params)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
ok like this

https://cs.opensource.google/go/go/+/refs/tags/go1.21.0:src/regexp/regexp.go;l=318-320